packages/lesswrong/components/sunshineDashboard/RejectedContentControls.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { registerComponent } from '../../lib/vulcan-lib/components';
+import { hasRejectedContentSectionSetting, isLWorAF } from '../../lib/instanceSettings';
+import RejectContentButton from './RejectContentButton';
+import RejectedReasonDisplay from './RejectedReasonDisplay';
+import { useDialog } from '../common/withDialog';
+import { DialogContent } from '../widgets/DialogContent';
+import LWDialog from '../common/LWDialog';
+import { highlightHtmlWithLlmDetectionScores } from './helpers';
+import { defineStyles, useStyles } from '../hooks/useStyles';
+
+const styles = defineStyles("RejectedContentControls", (theme: ThemeType) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+    gap: 8,
+    padding: 10,
+    borderRadius: 4,
+  },
+  llmScore: {
+    cursor: 'pointer',
+  },
+  automatedContentEvaluations: {
+    display: 'flex',
+    gap: 8,
+    alignItems: 'center',
+  },
+  aiOutput: {
+    fontSize: '0.9em',
+    textWrap: 'pretty',
+  },
+  row: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+  },
+}));
+
+interface ContentWrapper {
+  collectionName: 'Posts' | 'Comments';
+  content: any;
```

Comment:
this should not be `any`

Reply:
I wouldn't call this field `content`.  You could refactor to passing through the `contents` field of the post and/or comment (which would be reasonable), and then type it appropriately